ui/package.json

```diff
@@ -69,6 +69,7 @@
     "babel-jest": "^23.6.0",
     "babel-plugin-lodash": "^3.3.4",
     "babel-plugin-module-resolver": "^4.1.0",
+    "cypress-wait-until": "^1.7.2",
```

Comment:
I haven't found any real usage of `waitUntil`, it must be removed unless I am wrong.

Reply:
Admin Chronograf tab seems to have a lot of re-rendering going on as whenever I tried to use `click` the element would a lot of times be detached from the DOM, firing a Cypress error, while sometimes passing without a problem. This detaching would happen somewhere between selecting the element with `get` and clicking on the element with `click`, which only makes sense if the said element would be re-rendered. This is where `cypres-real-events` come into place as I used `realHover` to fire a hover event, which would allow Cypress to see a re-rendered element and click on it. This partially solved the issue, but after progressing through UI I found elements it doesn't work for. I was then recommended to read through Cypress Best Practices and Tips google doc, which had `clickAttached` in it with the following explanation

> This function asserts the element is attached first and uses jQuery to actually trigger the click so Cypress doesn't complain.

When it comes to `wait-until`, I'll remove that.